packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -235,17 +290,17 @@ class DraggableScrollableNotification extends Notification with ViewportNotifica
     required this.maxExtent,
     required this.initialExtent,
     required this.context,
-  }) : assert(extent != null),
-       assert(initialExtent != null),
-       assert(minExtent != null),
-       assert(maxExtent != null),
-       assert(0.0 <= minExtent),
-       assert(maxExtent <= 1.0),
-       assert(minExtent <= extent),
-       assert(minExtent <= initialExtent),
-       assert(extent <= maxExtent),
-       assert(initialExtent <= maxExtent),
-       assert(context != null);
+  })  : assert(extent != null),
```

Comment:
Make sure you get rid of the accidental formats before any final review.

IMO it's really silly and counter productive to have a major open source project without any automated formatting, but hey that's the call the Flutter team made so we just have to live with it. It's especially weird given that internally they have linters in their CI system that can block PRs and have pretty aggressive manual code style reviews on top of that.